lib/y2lan_restart_common.pm

```diff
@@ -1,3 +1,8 @@
+=head1 y2lan_restart_common
+
+YaST logic on Network Restart while no config changes were made
```

Comment:
What about, `Library for non-destructive testing using yast2 lan`

Reply:
Now update like this:

Library for non-destructive testing using yast2 lan. YaST logic on Network Restart while no config changes were made.